app/src/Api.tsx

```diff
@@ -113,6 +113,61 @@ export class FirebaseService {
       console.error(`Error completing task ${taskId}:`, error);
     }
   }
+
+  /**
+   * Get user's number of completed tasks by borough
+   * 
+   * @param userId 
+   * @param borough 
+   */
+  async getCompletedTasksByBorough(userId: string, borough: string): Promise<number | null> {
+    try {
+      borough = borough.toLowerCase();
+      const completedTasks = (await get(ref(this.db, `users/${userId}/${borough}_completed_tasks`))).val();
+      return !completedTasks ? 0 : Object.keys(completedTasks).length;
```

Comment:
Did we successfully remove that `default: 0` thing? Curious if there could be an off-by-1 error here with that

Reply:
yup the default doesn't show up anymore when I tested locally